src/payment/checkout/Checkout.jsx

```diff
@@ -13,9 +13,22 @@ import AcceptedCardLogos from './assets/accepted-card-logos.png';
 import PaymentForm from './payment-form/PaymentForm';
 import FreeCheckoutOrderButton from './FreeCheckoutOrderButton';
 import { PayPalButton } from '../payment-methods/paypal';
+import { AuthorizenetButton } from '../payment-methods/authorizenet';
 import { ORDER_TYPES } from '../data/constants';
 
 class Checkout extends React.Component {
+  handleSubmitAuthorizenet = () => {
```

Comment:
Can we use custom hook here containing authorizenet related functions/constants to minimize core changes ?